pages/00-Home.md

```diff
@@ -4,64 +4,18 @@ title: Foundations | Hydraecosystem.org
-
 hydrus is a set of **Python**-based tools for an easier and more efficient creation of hypermedia-driven REST APIs. hydrus utilises the power of [Linked Data](https://en.wikipedia.org/wiki/Linked_data) to create a powerful REST server to serve data.
 hydrus uses the [Hydra(W3C)](http://www.hydra-cg.com/) standard-to-be for creation and documentation of data servers.
-
-
-**Hydra smart client** (`hydra-python-agent`) is a Python implementation of a client based on the Hydra Community Group specifications and reference implementation [Heracles.ts](https://github.com/HydraCG/Heracles.ts). It can navigate a network of hydrus servers autonomously by reading their API documentation. It is a generic client because it can query and retrieve data from any, Hydra-aware or not, server (such as hydrus or any other Web server).
-
-### What makes the difference
-
-How does Hydra compare to other API documentation standards? Hydra is generally better rooted than other projects (i.e. OpenAPI) because it leverages the experience of the now thirty-years-old Linked Data ecosystem. This makes Hydra:
-* a 100% Semantic Web tool as requested by W3C; 
-* directly interoperable with any existing RDF/OWL repository ([have a look](https://en.wikipedia.org/wiki/Linked_data#Linking_Open_Data_community_project));
-* easily parsable into less fluent API documentation standards. As RDF is semantically highly descriptive, it is much easier to parse documents into a less rich standard flawlessly. While the opposite, such as parsing an OpenAPI document to make its API interoperable with existing Semantic repositories, can take a long time.
-
-These points highlight all the advantages mentioned in previous paragraphs. Finally, Linked Data is, by [Berners-Lee's design](https://www.w3.org/wiki/LinkedData), a tool to distribute data on the Web. Hydra adheres to this path of evolution to keep the Web open as wished by its creators and developers during the last few decades. 
-
-Table of contents
--------------
-* [Features](#features)
-* [Requirements](#req)
-* [Demo](#demo)
-* [Workflow](#workflow)
-* [Usage](#usage)
-* [Design](#design)
-
-<a name="features"></a>
-Features
--------------
+Besides all the other tools in the ecosystem, **hydrus** (lowercase) is the basic module from which API designers/engineers/developers can build up their own automated infrastructure to leverage smart clients. hydrus is the server-side tool that allows REST data to be published as Hydra-aware data. Hydrus provides a full-stack architecture that makes it the basic building block for a machine that is a node in a wider network of Web APIs.
```

Comment:
hydrus is neither a basic module nor server-side tool. Have you even read the description provided in the Hydrus repo?